packages/offix-datastore/src/predicates/PredicateFunctions.ts

```diff
@@ -0,0 +1,62 @@
+export abstract class PredicateFunction {
+    public filter(models: any[]) {
+        return models.filter((m) => this.evaluate(m));
+    }
+
+    public abstract evaluate(model: any): boolean;
+}
+
+export class ModelFieldPredicate extends PredicateFunction {
```

Comment:
Absolutely love this. That is what we support on the server. 
https://graphqlcrud.org/docs/next/find